platform/ios/app/MBXViewController.m

```diff
@@ -17,6 +17,8 @@
 #import "../src/MGLMapView_Experimental.h"
 
 #import <objc/runtime.h>
+#import <os/log.h>
+#import <os/signpost.h>
```

Comment:
iosapp is larded down and we’ve been trying to trim it back — since this logging functionality could presumably be useful in other/future view controllers, can we move this stuff out into a separate file?